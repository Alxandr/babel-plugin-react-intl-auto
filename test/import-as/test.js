import test from 'ava'
import macro from '../util'

test('import as', macro, __dirname)
