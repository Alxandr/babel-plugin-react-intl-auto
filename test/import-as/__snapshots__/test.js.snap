// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import as 1`] = `
"'use strict';

var _reactIntl = require('react-intl');

(0, _reactIntl.defineMessages)({
  hello: {
    defaultMessage: 'hello',
    'id': 'test/import-as/hello'
  },
  world: {
    defaultMessage: 'hello world',
    'id': 'test/import-as/world'
  }
});"
`;
